entries/c/consentmanager.net.json

```
@@ -0,0 +1,12 @@
+{
+  "Consent Manager": {
+    "domain": "consentmanager.net",
```

```suggestion
    "domain": "consentmanager.net",
    "url": "https://www.consentmanager.net/",
```

Because `https://consentmanager.net` redirects to their account portal (`app.consentmanager.net`), let's add a URL element.